exchanges/kraken/kraken_test.go

```diff
@@ -737,3 +738,624 @@ func TestWsCancelOrder(t *testing.T) {
 		t.Error(err)
 	}
 }
+
+func TestWsPong(t *testing.T) {
+	pressXToJSON := []byte(`{
+  "event": "pong",
+  "reqid": 42
+}`)
+	err := k.wsHandleData(pressXToJSON)
+	if err != nil {
+		t.Error(err)
+	}
+}
+
+func TestWsSystemStatus(t *testing.T) {
+	pressXToJSON := []byte(`{
+  "connectionID": 1333337,
```

Comment:
The API puts this at '8628615390848610000', can make the ConnectionID var use an int64

Reply:
That breaks 32 bit testing